InventoryManagement/InventoryManagement/Models/Product.cs

```diff
@@ -5,8 +5,15 @@ namespace InventoryManagement.Models
     public class Product
     {
         public int Id { get; set; }
+
+        [Required(ErrorMessage = "UPC is required")]
+        [StringLength(20, ErrorMessage = "UPC cannot exceed 20 characters")]
         public string UPC { get; set; } = string.Empty;
+
+        [Required(ErrorMessage = "Title is required")]
+        [StringLength(100, ErrorMessage = "Title cannot exceed 100 characters")]
```

Comment:
Two comments:
1) This was suggested in a previously submitted code review.  When that occurs, only the first in gets credit.
2) This is a very good suggestion, but go further.  There are many more attributes that can use validation at the model level.